package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "src",
+  "version": "1.0.0",
+  "description": "---",
+  "main": "router.js",
```

Comment:
this should default to `index.js`

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#main